app-text/cutemarked/cutemarked-0.11.3.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="Qt Markdown editor"
+DESCRIPTION="A Qt-based, free and open source Markdown editor with live HTML \
+preview, math expressions, code and markdown syntax highlighting."
```

Comment:
What do you think of this variant?
```diff
-SUMMARY="Qt Markdown editor"
+SUMMARY="A Qt-based Markdown editor"
-DESCRIPTION="A Qt-based, free and open source Markdown editor with live HTML \
-preview, math expressions, code and markdown syntax highlighting."
+DESCRIPTION="CuteMarkEd is a Qt-based Markdown editor with live HTML preview, \
+math expressions, code and markdown syntax highlighting."
```
All software in haikuports is free and open source, so we don't really need to mention it.